core/src/mindustry/core/NetServer.java

```diff
@@ -98,6 +98,15 @@ public class NetServer implements ApplicationListener{
     private boolean closing = false, pvpAutoPaused = true;
     private Interval timer = new Interval(10);
     private IntSet buildHealthChanged = new IntSet();
+    
+    /** Current kick sessions. */
+    public VoteSession[] currentlyKicking = {null};
```

Comment:
There is not reason to make this an array if it's not a local variable. Also, mark it `@Nullable.`

Reply:
Oops, didn't realize it was an array only to allow other code to modify it